Formula/f/foldseek.rb

```diff
@@ -0,0 +1,50 @@
+class Foldseek < Formula
+  desc "Search your protein structures in seconds"
+  homepage "https://github.com/steineggerlab/foldseek"
+  url "https://github.com/steineggerlab/foldseek/archive/8-ef4e960.tar.gz"
+  version "8-ef4e960"
+  sha256 "c74d02c4924d20275cc567783b56fff10e76ed67f3d642f53c283f67c4180a1e"
+  license "GPL-3.0-or-later"
+  head "https://github.com/steineggerlab/foldseek.git", branch: "master"
+
+  depends_on "cmake" => [:build, :test]
+  depends_on "rust" => :build
+
+  uses_from_macos "bzip2"
+  uses_from_macos "zlib"
+
+  on_macos do
+    depends_on "libomp"
+  end
+
+  resource "homebrew-testdata" do
+    url "https://raw.githubusercontent.com/steineggerlab/foldseek/master/example/d1asha_"
+    sha256 "b4ec14f5decc94b5363b3414db4d25e3e09039c7a6fbb585041730dcf3cc1fd8"
+  end
+
+  def install
+    # fix "DSO missing from command line" error
+    ENV.append "LDFLAGS", "-Wl,--copy-dt-needed-entries" if OS.linux?
+    args = *std_cmake_args
+    if OS.mac?
+      libomp = Formula["libomp"]
+      args << "-DOpenMP_C_FLAGS=-Xpreprocessor -fopenmp -I#{libomp.opt_include}"
+      args << "-DOpenMP_C_LIB_NAMES=omp"
+      args << "-DOpenMP_CXX_FLAGS=-Xpreprocessor -fopenmp -I#{libomp.opt_include}"
+      args << "-DOpenMP_CXX_LIB_NAMES=omp"
+      args << "-DOpenMP_omp_LIBRARY=#{libomp.opt_lib}/libomp.a"
+    end
```

Comment:
Can the detection of libomp be fixed upstream, so we don't need all these settings?

Reply:
It's not impossible, probably. But, I would like to ask if it's really necessary. For example, https://github.com/Homebrew/homebrew-core/blob/92c6769d6d4cce1026d8db5c424014d198bc60e1/Formula/g/gismo.rb seems passed with a similar tweak.

Reply:
Yes: https://docs.brew.sh/Acceptable-Formulae#niche-or-self-submitted-stuff

Reply:
I see. I withdraw this formula and transfer it to brewsci/bio repo. Thanks, @SMillerDev.